src/shared/lib/analytics/AnalyticsService.ts

```diff
@@ -39,6 +39,7 @@ export const MixEvents = {
   UploadLogsPressed: 'Upload Logs Pressed',
   UploadedLogsSuccessfully: 'Uploaded Logs Successfully',
   UploadLogsError: 'Upload Logs Error Occurred',
+  NotificationTap: 'Notification tap',
```

Comment:
Maybe update to `Notification tapped` to match the copy style from the other events?

Reply:
@binho this is the expected name according to the ticket